### DISCUSSION TITLE
Common Mode Voltage on Port 1 & 2

### DISCUSSION THREAD

**Discussion by callendorph** [jankae/LibreVNA#272] [Q&A] [answered]
The circuit I want to measure has a DC bias. I plan to apply this DC bias in a slow ramp to the DUT.

If I look at the [schematic](https://github.com/jankae/LibreVNA/blob/master/Hardware/Schematic.pdf) on pages 6 and 7 at capacitors C99 and C136, respectively, the BOM lists a [ATC530L](https://www.mouser.com/datasheet/2/586/530l-1384710.pdf?srsltid=AfmBOopHSQo9ivXOqU6wYzZtP-CQ4ZdRXVjB5lQvqFedO-_OH9xjvTU9) series or compatible capacitor. 

That capacitor series reports a 16V DC rating. So it seems like that capacitor could handle a bit of DC bias but nothing crazy. 

Is there anything else I should be worried about when including a DC bias to the DUT?

It seems like I could replace that capacitor with a higher voltage rated capacitor, such as [PPI 0402BB104](https://passiveplus.com/wp-content/uploads/2024/05/PPI-0402BB104KW500-DATASHEET-050824RevA.pdf), and then it would be able to withstand a slightly higher DC bias. Do you agree with that or is there something else I should consider ?

**Comment by jankae** [comment, marked as answer]
Yes, that capacitor limits the DC voltage and replacing it would potentially allow you to use higher voltages.

But I wouldn't bother soldering here. The usual way is to attach a bias tee to the port. This allows you to apply a DC bias which is isolated from the VNA.

If you don't have a bias tee, an external DC block might also do the trick. But if you don't have that as well, your proposed solution might be worth a try...

**Comment by callendorph** [comment]
Good point - the bias T might be a better path forward.
